packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -21,6 +28,12 @@ interface BaseDatePickerProps {
    */
   readonly selected?: Date;
 
+  /**
+   * Determines if the focus moves to the selected date (if any) or back to
+   * the activator
+   */
+  readonly smartAutofocus?: boolean;
```

Comment:
Wondering if this needs to be optional, since it's always either true or false in inputdate or datepicker

Reply:
@adambobadam let's say it's not optional. What would it do to the component when you do start using them?

Reply:
Ahh, I guess what I was asking is if it's a necessary prop to expose since it's just hard-coded as true and false in two places and we don't use it anywhere else inside datepicker and inputdate

Reply:
In atlantis, yeah. Who knows how many places DatePicker would be used outside Atlantis. So if this is required, you'd always have to declare `smartAutofocus` on every usage. 

As opposed to being optional and set it to true, always 
![image](https://user-images.githubusercontent.com/15986172/146967903-de7231b7-2e49-4bf0-bb44-8d7f6f6ac3c5.png)

Until the consumer says, y'know what, i don't want to